plugins/GSdx/GSSettingsDlg.cpp

```diff
@@ -389,6 +389,7 @@ void GSSettingsDlg::UpdateControls()
 		bool ocl = renderer == GSRendererType::DX9_OpenCL || renderer == GSRendererType::DX1011_OpenCL || renderer == GSRendererType::OGL_OpenCL;
 		bool null = renderer == GSRendererType::Null;
 
+		// GSdx logos
```

Comment:
Isn't this kinda obvious from looking at the IDC identifiers?

Same applies to quite a few of the other comments.